### ISSUE TITLE
array.isEvery does not check for true by default

### ISSUE THREAD

**Issue by mwpastore** [kellyselden/ember-awesome-macros#342]
The second argument to Ember.Array#isEvery defaults to `true` if not specified. It would be nice if `array.isEvery` did the same.

```javascript
array: Ember.A([{ test: true }, { test: true }]),
key: 'test',
result: array.isEvery('array', 'key'), // false (expected: true)
```

**Comment by kellyselden** [comment]
Resolved by #364

**Comment by mwpastore** [comment]
Thank you!
